frontend/src/metabase/components/CopyTextInput/CopyTextInput.tsx

```diff
@@ -11,7 +11,7 @@ const defaultProps = {
 };
 
 export const CopyTextInput = forwardRef(function CopyTextInput(
-  props: TextInputProps,
+  props: TextInputProps & { value: string },
```

Comment:
`SecretKeyModal` is the only component that uses this component, and the `value` passed in is typed as `string`. The `TextInputProps` value is `string | undefined`, which would cause type errors for the `CopyButton`.